frontend/homework/10/Todolist.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+
+import Header from './Header';
+import Midsection from './Midsection';
+
+function Todolist() {
+
+  const [searchTerm, setSearchTerm] = useState<string>('');
+  const [todos, setTodos] = useState<string[]>([]);
+  
+  const [newTodoText, setNewTodoText] = useState<string>('');
+
+  return (
+    <div>
+      <Header searchTerm={searchTerm} setSearchTerm={setSearchTerm}/>
+      <Midsection
+        searchTerm={searchTerm}
+        setSearchTerm={setSearchTerm}
+        newTodoText={newTodoText}
+        setNewTodoText={setNewTodoText}
+        todos={todos}
+        setTodos={setTodos}
```

Comment:
Antipattern - prop plowing - solve this by putting all props in a single object and passing that object instead. Like `<MidSection props={..props}>`